tests/tests/connectors/test-class-connector-acf.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace WP_Stream;
```

Comment:
I am not certain how @kasparsd feels, but I would like to see some more documentation in these tests.

Reply:
There was a doc block, but it was lost in PR somewhere. I'll correct this.

Reply:
This has been [fixed](https://github.com/xwp/stream/pull/1118/files#diff-8b4d036afce8fdd07963c1eba24744a1R2-R6), but the snippet isn't updated :thinking: